DSCResources/MSFT_xPowerPlan/MSFT_xPowerPlan.psm1

```diff
@@ -0,0 +1,108 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [ValidateSet('Yes')]
+        [System.String]
+        $IsSingleInstance,
```

Comment:
Doesnt look like you are doing anything with $IsSingleInstance. What is its purpose? Can you please add some comments? Thanks!